ddtrace/contrib/pyramid/trace.py

```diff
@@ -61,6 +63,8 @@ def trace_tween_factory(handler, registry):
     tracer = settings.get(SETTINGS_TRACER) or ddtrace.tracer
     enabled = asbool(settings.get(SETTINGS_TRACE_ENABLED, tracer.enabled))
     distributed_tracing = asbool(settings.get(SETTINGS_DISTRIBUTED_TRACING, True))
+    analytics = asbool(settings.get(SETTINGS_ANALYTICS, False)) if settings.get(SETTINGS_ANALYTICS) else None
```

Comment:
Do we need `asbool` here?

Reply:
If `SETTINGS_ANALYTICS` is in `settings`, then calling `settings.get(SETTINGS_ANALYTICS, False)` should never hit the default value.

Can probably reduce this all to:

```python
analytics = settings.get(SETTINGS_ANALYTICS)
```

Default will be `None` which is what you want. You are calling `asbool()` when parsing env variable, so if they choose to set as a non-bool manually, that is on them.